subcommands/targets/image.go

```diff
@@ -27,13 +27,13 @@ func doImage(cmd *cobra.Command, args []string) {
 	inputTarget := args[0]
 	logrus.Debugf("Generating image of Target %s in Factory %s", inputTarget, factory)
 
-	url, err := api.TargetImageCreate(factory, inputTarget)
+	JobServUrl, WebUrl, err := api.TargetImageCreate(factory, inputTarget)
```

Comment:
this looks good. just one nit. Can you make this `jobServUrl, webUrl`. It seems a bit unusual in golang to name a local variable with a capital letter.

Reply:
absolutely agree.

Reply:
done